source/installation/howto-web-no-local-install.md

```diff
@@ -7,9 +7,7 @@ Let's see how to use OpenFisca on one of those services: [repl.it](https://repl.
 
 ## Instructions
 
-1. Go to [repl.it](https://repl.it) and select `Python`:
-
-    [![repl.it homepage](https://raw.githubusercontent.com/openfisca/openfisca-doc/master/source/static/img/replit-homepage.png)](https://github.com/openfisca/openfisca-doc/blob/master/source/static/img/replit-homepage.png)
```

Comment:
Thanks for removing those obsolete screenshots!
Could you please remove the files as well? 🙂

Reply:
Done ^^